From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Tue, 28 Jul 2020 06:13:11 +1200
Subject: Reduce commented-out Ubuntu packages for C# in chroot to just
 "mono-runtime"

mono-mcs (the compiler) is only required outside the chroot.
mono-devel pulls in a lot of unnecessary packages, including mono-mcs.

This change makes the Ubuntu and Debian versions consistent.
---
 misc-tools/dj_make_chroot.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc-tools/dj_make_chroot.in b/misc-tools/dj_make_chroot.in
index eefe818ac..019f8895c 100755
--- a/misc-tools/dj_make_chroot.in
+++ b/misc-tools/dj_make_chroot.in
@@ -163,7 +163,7 @@ INCLUDEDEBS=""
 
 # Packages to install after upgrade (space separated):
 INSTALLDEBS="default-jre-headless pypy python3 locales"
-# For C# support add: mono-mcs mono-devel
+# For C# support add: mono-runtime
 # However running mono within chroot still gives errors...
 
 # Packages to remove after upgrade (space separated):
-- 
2.20.1

